src/Sylius/Bundle/AddressingBundle/Behat/AddressingContext.php

```diff
@@ -141,14 +142,17 @@ public function thereIsProvince($name)
      */
     public function theFollowingCountryTranslationsExist(TableNode $table)
     {
-        $manager = $this->getEntityManager();
+        $manager = $this->getManager('country');
 
         foreach ($table->getHash() as $data) {
             $countryTranslation = $this->findOneByName('country_translation', $data['country']);
             $country = $countryTranslation->getTranslatable();
             $country
                 ->setCurrentLocale($data['locale'])
-                ->setName($data['name']);
+                ->setName($data['name'])
+            ;
+
+            $manager->persist($country);
```

Comment:
Don't need to call ``` persist ``` on entities which already are in identity map (in this case are loaded from repository).